www/web/src/components/FeatureCard.astro

```diff
@@ -13,11 +13,11 @@ const { icon, description, feature } = Astro.props
 <div
   class='animate-border rounded-lg border border-transparent [background:linear-gradient(45deg,#172033,theme(colors.slate.800)_50%,#172033)_padding-box,conic-gradient(from_var(--border-angle),theme(colors.slate.600/.48)_80%,_theme(colors.indigo.500)_86%,_theme(colors.indigo.300)_90%,_theme(colors.indigo.500)_94%,_theme(colors.slate.600/.48))_border-box]'
 >
-  <div class='h-full place-self-center rounded-lg bg-[--purple-900] p-10'>
-    <div class='flex items-center justify-center gap-2 pb-6'>
+  <div class='h-full place-self-center rounded-lg bg-[--slate-100] dark:bg-[--purple-900] p-10'>
```

Comment:
--slate-100 is not a defined variable in `globals.css` it should be updated to `bg-slate-100`